src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -1169,26 +1175,36 @@ static String getRanges(JsonArray ar, String startProp, String endProp) {
       LongAdder [] uniqueItemRequestsByPeriod = LongAdder.arrayOfLength(accessCountPeriods.size());
       rowSet.forEach(row -> {
         JsonArray accessCountsByPeriod = new JsonArray();
+        LongAdder accessCountTotal = new LongAdder();
         boolean unique = "Unique_Item_Requests".equals(row.getString("metrictype"));
+        int pos0 = journal ? 6 : 5;
```

Comment:
I missed the `7` in the previous version, but these magic numbers make me nervous. Can we make them named constants?

Reply:
Changed.

Reply:
Appreciated.